pytorch_blade/.bazelversion

```diff
@@ -0,0 +1 @@
+4.2.2
```

Comment:
5.0.0 since the rebase commit (20220216) is merged.

Reply:
done